docker/core/http/pep_config.js

```diff
@@ -0,0 +1,79 @@
+const config = {};
+
+function toBoolean(env, defaultValue) {
+  return env !== undefined ? env.toLowerCase() === 'true' : defaultValue;
+}
+
+function to_array(env, default_value){
+    return (env !== undefined) ? env.split(',') : default_value;
+}
+
+// Used only if https is disabled
+config.pep_port = process.env.PEP_PROXY_PORT || 80;
+
+// Set this var to undefined if you don't want the server to listen on HTTPS
+config.https = {
+  enabled: toBoolean(process.env.PEP_PROXY_HTTPS_ENABLED, false),
+  cert_file: 'cert/cert.crt',
+  key_file: 'cert/key.key',
+  port: process.env.PEP_PROXY_HTTPS_PORT || 443,
+};
+
+config.idm = {
```

Comment:
[minor] it's generic comments, write proper comments on like Config.idm, config.app, cache time explain one linear before all the configuration attributes.

Reply:
added comments